index.html

```diff
@@ -0,0 +1,74 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="styles.css">
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+    <!--
+
+    Hello Camper!
+
+    For now, the test suite only works in Chrome!
+    Please read the README below in the JS Editor before beginning.
+    Feel free to delete this message once you have read it.
+    Good luck and Happy Coding!
+
+    - The freeCodeCamp Team
+
+    -->
+    <main id="main-doc">
+      <section class="main-section" id="Introduction">
+        <header>Introduction</header>
+        <p></p>
+        <p></p>
```

Comment:
- [ ] [ Required ] Kindly I suggest you that do not use the empty `p` tag. 😄

Reply:
[ ] Kindly try to always include an explanation, a "why", for each requested change. For example:`Why don't you use an article tag for the .first-article element instead of a div, that would improve the HTML structure and make the page more accessible

Reply:
Ok thank you but I confuse should I add this things again in comments?

Reply:
[ ] Kindly try to add an article for your `p` tags that would improve the HTML structure and make the page more accessible.